### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by amagana3**
### Overview

- Add create LTK endpoint
- Add delete LTK endpoint
- Unit tests

### CODE REVIEW

**Review by ntangy — APPROVED** [review]

**Comment by webbbarker** [comment]
Let me have a few minutes to review before we merge this morning if you don't mind.

**Review by webbbarker — CHANGES_REQUESTED** [review]
I've got several changes that need to be rolled into `develop` before we can cut the master release.  Let's go ahead and cancel this one for now and I will get a new PR out with those fixes (since I cannot reopen a closed and merged PR).

**Pull request closed without merging**
